stan-dev/stan
# cpplint warnings: removed all from src/stan/math

#### Summary:

Removes all cpplint warnings from `src/stan/math`.

#### Intended Effect:

**Note: this does not change code behavior.**

Removes cpplint warnings by:
- changing casts
- changing whitespace
- adding braces where necessary (`{` and `}`)
- in a very few places, added `// NOLINT` to disable cpplint rules

#### How to Verify:

The tests haven't changed, so passing all tests show that it works as intended.

The way to verify this is by eye, but it should be good if it passes tests. Someone should spot-check some of the changes by hand, though.

#### Side Effects:

This may affect existing pull requests. If this is ok, I'll handle the merges.

#### Documentation:

None.

#### Reviewer Suggestions: 

Anyone.
